content/faq/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Frequently Asked Questions"
+weight: 5
+---
+
+## Using NWB
+
+
+### Is NWB 2 stable? {#is-nwb-2-stable}
+
+Yes! NWB 2.0 was officially released in January 2019, and the schema is stable. A key goal of the NWB endavour is to ensure that NWB 2 remains accessible. As NWB evolves we strive to ensure that any changes we make do not break backwards compatiblility.
+
+### I would like to use NWB. How do I get started? {#how-to-get-started}
+
+See the [Converting neurophysiology data to NWB](/converting-data-to-nwb/) page for more information.
+
+### How do I cite NWB 2 in my research? {#how-to-cite}
+
+Oliver Rübel, Andrew Tritt, Ryan Ly, Benjamin K. Dichter, Satrajit Ghosh, Lawrence Niu, Pamela Baker, Ivan Soltesz, Lydia Ng, Karel Svoboda, Loren Frank, Kristofer E. Bouchard, The Neurodata Without Borders ecosystem for neurophysiological data science, Oct. 2022, eLife 11:e78362. [https://doi.org/10.7554/eLife.78362](https://doi.org/10.7554/eLife.78362)
+
+### How do I install PyNWB? {#install-pynwb}
+
+See the [Installing PyNWB](https://pynwb.readthedocs.io/en/stable/install_users.html) for details.
+
+### How do I install MatNWB? {#install-matnwb}
+
+See the [MatNWB documentation](https://matnwb.readthedocs.io/en/latest/pages/getting_started/installation_users.html) for details.
+
+### What is the difference between PyNWB and nwb-schema? {#pynwb-vs-schema}
+
+[PyNWB](https://pynwb.readthedocs.io/en/stable/) is the Python reference read/write API for the current NWB 2.x format. The [nwb-schema](https://github.com/NeurodataWithoutBorders/nwb-schema/) is used to manage development of the data standard schema. End-users who want to use NWB typically do not need to worry about the nwb-schema repo as the current schema is always installed with the corresponding API (whether it is [PyNWB](https://pynwb.readthedocs.io/en/stable/) for Python or [MatNWB](https://matnwb.readthedocs.io/en/latest/) for Matlab).
+
+### How do I read NWB files in different programming languages? {#read-nwb-files}
+
+For Python and Matlab we recommend using the [PyNWB](https://pynwb.readthedocs.io/en/stable/) and [MatNWB](https://matnwb.readthedocs.io/en/latest/) reference APIs. To get started see also the [Reading NWB Files](https://nwb-overview.readthedocs.io/en/latest/file_read/file_read.html) page.
```

Comment:
For C++ I would add the link to the AqNWB read page https://neurodatawithoutborders.github.io/aqnwb/read_page.html